src/Features/SupportConsoleCommands/Commands/Upgrade/UpdateForgetComputedCalls.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Livewire\Features\SupportConsoleCommands\Commands\Upgrade;
+
+use Livewire\Features\SupportConsoleCommands\Commands\UpgradeCommand;
+
+class UpdateForgetComputedCalls extends UpgradeStep
+{
+    public function handle(UpgradeCommand $console, \Closure $next)
+    {
+        $this->interactiveReplacement(
+            console: $console,
+            title: 'The forgetComputed component method is replaced by php\'s unset function',
+            before: '$this->forgetComputed()',
+            after: 'unset()',
+            pattern: '/\$this->forgetComputed\([\'"](.+)[\'"]\)/',
+            replacement: 'unset($this->$1)',
+            directories: ['app'],
+            subjectName: 'calls'
```

Comment:
This doesn't exist in this method.

Reply:
Thanks. Now it does. I've added this to improve the message for the user. It now states this:
```This means all $this->forgetComputed() calls must be changed to unset().``` instead of 
```This means all $this->forgetComputed() directives must be changed to unset().```